src/rs485/protocols/checksums/modbus_rtu.cpp

```diff
@@ -0,0 +1,43 @@
+#include "rs485/protocols/checksums/modbus_rtu.h"
+
+void ModbusRTUChecksum::add(uint8_t data) {
+  uint16_t temp = (data ^ checksum) & 0xff;
+  checksum >>= 8;
+  checksum ^= crcTable[temp];
+  checksum &= 0xffff;
+}
+
+const uint16_t ModbusRTUChecksum::crcTable[] = {
```

Comment:
Where do the values in this table come from? Could you leave a link to some reference here?